verilog/tools/ls/verilog-language-server_test.cc

```diff
@@ -580,6 +580,19 @@ std::string DefinitionRequest(absl::string_view file, int id, int line,
   return formattingrequest.dump();
 }
 
+// Creates a textDocument/definition request
```

Comment:
This comment looks like it has been copy/pasted from somewhere :)
It also indicates that we might have an opportunity to combine these.

Reply:
Yes, I updated the comment, used `verible::lsp`-based structure and moved most of the implementation to a common function used by `textDocument/references` and `textDocument/definition`.